src/components/DriverCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function DriverCard(props) {
+    const {name, rating, img, car} = props;
+
+        function stars(rating){
+            if (rating < 0.5) return "☆☆☆☆☆"
+            else if (rating < 1.5) return "★☆☆☆☆"
+            else if (rating < 2.5) return "★★☆☆☆"
+            else if (rating < 3.5) return "★★★☆☆"
+            else if (rating < 4.5) return "★★★★☆"
+            else return "★★★★★"
```

Comment:
Instead of copying the function from the Rating Component and pasting it here. You can and should re-use the Rating Component:

```
import React from 'react';
import Rating from './Rating';

function DriverCard(props) {
  const { name, rating, img, car } = props;

  return (
    <div>
      <img className="imageLicense" src={img} alt="" />
      <Rating>{rating}</Rating>
      <p>{name}</p>
      <p>
        {car.model} - {car.licensePlate}
      </p>
    </div>
  );
}

export default DriverCard;
```